src/server/json_family.cc

```diff
@@ -1466,7 +1466,7 @@ void RecursiveMerge(const JsonType& patch, JsonType* dest) {
   for (const auto& k_v : patch.object_range()) {
     if (k_v.value().is_null()) {
       dest->erase(k_v.key());
-    } else {
+    } else if (dest->find(k_v.key()) != dest->object_range().end()) {
```

Comment:
i suspect it's not a correct fix. Above this function there is a spec with a pseudo code. Have you looked at it?

Reply:
yes but the pseudocode is not very clear. I will need to think about it but I will take care of this.